module_auto_update/addon_hash.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 ACSONE SA/NV.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+import hashlib
+import os
+from fnmatch import fnmatch
+
+
+def _fnmatch(filename, patterns):
+    for pattern in patterns:
+        if fnmatch(filename, pattern):
+            return True
+    return False
+
+
+def _walk(top, exclude_patterns, keep_langs):
+    keep_langs = {l.split("_")[0] for l in keep_langs}
```

Comment:
flake8 except __init__.py................................................Failed
- hook id: flake8
- duration: 0.89s
- exit code: 1
module_auto_update/addon_hash.py:17:39: E741 ambiguous variable name 'l'

Reply:
@atchuthan Issue has been adressed and fixed.